### PR TITLE
DOCS-716 asses onboarding docs + extra nits

### PR DESCRIPTION
**Pull request by andf-viam**
Small tweaks to low-hanging fruit where discussion or further consideration is unlikely to be necessary. Kept most changes to within onboarding sections -- Viam in 3 Minutes, Try Viam, and the Installation Guide -- but my monospace efforts for `viam-server` ended up being corpus wide 😅 .

Happy for any corrections if I did something wrong -- thanks!!

### CODE REVIEW

File: docs/try-viam/rover-resources/_index.md
Comment (andf-viam) [inline docs/try-viam/rover-resources/_index.md:18]: Personal nit. "order" or "buy" followed by "now" is a sales exhortation. Either preceded by "now" is instead a "recently-made-available" advisement. Obviously feel free to disregard this if I am being too pedantic 😅 .

File: docs/try-viam/rover-resources/rover-tutorial.md
Comment (sguequierre) [inline docs/try-viam/rover-resources/rover-tutorial.md:155]: nice

**Review by sguequierre — APPROVED** [review]
wonderful catches, woohoo! I don't know if any of these changes require @npentrel's approval but I'll go ahead and approve if you want to get these fixes in. 
Nice work!

File: docs/appendix/glossary/client-application.md
Comment (npentrel) [inline docs/appendix/glossary/client-application.md:11]:
```suggestion
You can run a client application on the same {{< glossary_tooltip term_id="part" text="part" >}} that runs `viam-server`, or on a separate device.
```

**Review by npentrel — APPROVED** [review]

**Pull request merged**
